integration/sunbird/compare_sunbird_with_glpi.py

```diff
@@ -192,14 +192,17 @@ def get_sunbird_machines(
 
     sunbird_machines = {}
     for location in config_map:
+        cabinets = config_map.get(location, {}).get("Cabinets") or {}
+        if isinstance(cabinets, dict):
+            cabinets = cabinets.values()
```

Comment:
Believe we need to cast this to a list for it to work.
```suggestion
            cabinets = list(cabinets.values())
```

Reply:
Didn't break one way or the other for me so 👍